src/std_nodes/file_append_bolt.ts

```diff
@@ -248,3 +249,75 @@ export class FileAppendBolt implements intf.Bolt {
         }
     }
 }
+
+
+/** This bolt writes incoming messages to file in CSV format. */
+export class CsvFileAppendBolt implements intf.Bolt {
+
+    private transform: TransformHelper;
+    private file_name: string;
+    private delimiter: string;
+    private current_data: string[];
+
+    constructor() {
+        this.file_name = null;
+        this.transform = null;
+        this.current_data = [];
+    }
+
+    init(_name: string, config: any, _context: any, callback: intf.SimpleCallback) {
+        this.file_name = config.file_name;
+        this.delimiter = config.delimiter || ",";
+        if (config.delete_existing) {
+            if (fs.existsSync(this.file_name)) {
+                fs.unlinkSync(this.file_name);
```

Comment:
this may still fail

Reply:
My concern is that if it fails it will throw and the callback will not be called - we need a try/catch

Reply:
good point 😉 
will add it